source/codegen/metadata/nirfmxspecan/config.py

```diff
@@ -58,4 +58,5 @@
     'metadata_version': '0.1',
     'module_name': 'nirfmxspecan',
     'session_class_description': 'An NI-RFmxSpecAn instrument handle',
+    'disallow_session_reverse_lookup': True
```

Comment:
This will cause generation to fail if there's any method that has an output session from the driver API that's not marked as a `init_method`. Essentially, this scenario would be broken for RFmx because there is no mapping currently between a session handle and grpc session since it can be one to many.

I added this to RFSG temporarily and confirmed it caused generation to fail for RFSG's `GetAttributeViSession` as expected.